docs/patterns/existing-eks-observability-accelerators/existing-eks-apiserver-observability.md

```diff
@@ -0,0 +1,162 @@
+# Existing EKS Cluster apiserver Observability Accelerator
```

Comment:
rename this to Existing EKS Cluster Observability Accelerator - API Server monitoring

Reply:
Done